apps/costcontrol/js/views/appdetail.js

```diff
@@ -82,7 +82,11 @@ var AppDetailView = (function() {
                 enabled: true
               }
             },
-            todayLabel: {}
+            todayLabel: {},
+            accessibility: {
+              dataLimitValue: settings.dataLimitValue,
```

Comment:
IMO, these fields should be allocated under model.limits section, it makes the fields easily accessible.